framework/src/variables/MooseVariableFE.C

```diff
@@ -408,6 +408,73 @@ MooseVariableFE<OutputType>::computeElemValuesFace()
   _element_data->computeValues();
 }
 
+template <typename OutputType>
+void
+MooseVariableFE<OutputType>::computeFaceValues(const FaceInfo & fi)
+{
+  _element_data->setGeometry(Moose::Face);
+  _neighbor_data->setGeometry(Moose::Face);
+  const auto facetype = fi.faceType(std::make_pair(this->number(), this->sys().number()));
+  if (facetype == FaceInfo::VarFaceNeighbors::NEITHER)
+    return;
+  else if (facetype == FaceInfo::VarFaceNeighbors::BOTH)
+  {
+    _element_data->computeValuesFace(fi);
+    _neighbor_data->computeValuesFace(fi);
+  }
+  else if (facetype == FaceInfo::VarFaceNeighbors::ELEM)
+  {
+    _element_data->computeValuesFace(fi);
+    _neighbor_data->computeGhostValuesFace(fi, *_element_data);
+  }
+  else if (facetype == FaceInfo::VarFaceNeighbors::NEIGHBOR)
+  {
+    _neighbor_data->computeValuesFace(fi);
+    _element_data->computeGhostValuesFace(fi, *_neighbor_data);
+  }
+  else
+    mooseError("FE to FV coupling broken.");
```

Comment:
```suggestion
    mooseAssert(false, "FE to FV coupling broken.");
```
We dont expect to hit this in production